homeassistant/helpers/llm.py

```diff
@@ -496,7 +496,7 @@ def _get_exposed_entities(
         CALENDAR_DOMAIN: {},
     }
 
-    for state in hass.states.async_all():
+    for state in sorted(hass.states.async_all(), key=lambda state: state.name):
```

Comment:
```suggestion
    for state in sorted(hass.states.async_all(), key=attrgetter("name")):
```

Reply:
https://docs.python.org/3/library/operator.html#operator.attrgetter

You'll need to import it as well